src/engine/segment.go

```diff
@@ -99,7 +99,8 @@ const (
 	// ANGULAR writes which angular cli version us currently active
 	ANGULAR SegmentType = "angular"
 	// AWS writes the active aws context
-	AWS SegmentType = "aws"
+	ARGOCD SegmentType = "argocd"
```

Comment:
Can you add the same (I know) comment for consistency?

Reply:
fixed with the correct comment.